src/api/liveSpecsExt.ts

```diff
@@ -331,11 +332,10 @@ const getLiveSpecsByCatalogNames = async (
         index = index + CHUNK_SIZE;
     }
 
-    const res = await Promise.all(promises);
-
-    const errors = res.filter((r) => r.error);
-
-    return errors[0] ?? res[0];
+    const responses = await Promise.all(promises);
+    return parsePagedFetchAllResponse<LiveSpecsExtQuery_ByCatalogNames>(
+        responses
+    );
```

Comment:
This was dumb. Many times when we make large paginated calls we do not need the full response. This is NOT one of the cases. So now we always return the full set of errors and data back.

This is the fix for issue `2` of https://github.com/estuary/ui/issues/1741

Reply:
Using the shared function here even though we aren't using the `pagedFetcher` as that switch over is a lot more complex.